sklearn/mixture/gmm.py

```diff
@@ -394,7 +394,7 @@ def sample(self, n_samples=1, random_state=None):
         if random_state is None:
             random_state = self.random_state
         random_state = check_random_state(random_state)
-        weight_cdf = np.cumsum(self.weights_)
+        weight_cdf = stable_cumsum(self.weights_)
```

Comment:
Again, I think this is unlikely to be a problem case. `self.weights_.shape` is small.